betterproto/__init__.py

```diff
@@ -866,7 +900,9 @@ def from_dict(self: T, value: dict) -> T:
                     elif meta.wraps:
                         setattr(self, field_name, value[key])
                     else:
-                        v.from_dict(value[key])
+                        v = v.from_dict(value[key])
+                        if v is not None:
+                            setattr(self, field_name, v)
```

Comment:
I'm a bit confused how `from_dict` ever worked properly, if message fields were not assigned to their parent before..

any idea of why this was needed, or why `from_dict` did actually work?

Also, lines `930`-`931` assign `v` to the field. Perhaps just assigning `v=` would suffice?

Reply:
Ah, I realized it!

`v` is an existing Message object. The required behavior is that the data from the map is _merged_ into the existing fields. I think no assignment is needed here.

Reply:
Yeah correct, the `v = v.from_dict(value[key])` isn't needed, I'm just used to transforming objects vs mutating them that I wrote it like that! Also, it looks like I can just move lines `930`-`931` out an indentation layer and achieve the same effect 😄